Modules/trigger.py

```diff
@@ -0,0 +1,33 @@
+import pydle
+
+
+class Trigger(object):
+    """Object to hold information on the user who invoked a command (and where they did it)."""
+
+    def __init__(self, bot: pydle.BasicClient, nickname: str, target: str, ident: str, hostname: str,
+                 realname: str = "", away: str = None, account: str = None, identified: bool = False):
+        self.bot = bot
+        self.nickname = nickname
+        self.target = target
+        self.ident = ident
+        self.hostname = hostname
+        self.realname = realname
+        self.away = away
+        self.account = account
+        self.identified = identified
+
+    @classmethod
+    def from_bot_user(cls, bot: pydle.BasicClient, nickname: str, target: str):
+        """Creates a `Trigger` object from a user dictionary as used by pydle."""
```

Comment:
please place the parameters in the docstring. also, methinks a type hint could be nice here ;)

Reply:
Are the names bot, nickname and target not self-explanatory? Anything more than a one-liner strikes me as a waste of space here.

Reply:
23:09 <&xlexious> we are not programming on a fucking bbc micro with 4kb of memory

Reply:
By your logic Unit tests are a waste of space.
By your logic all documentation is a waste of space.

Reply:
23:09 <&xlexious> we are not programming on a f*****g bbc micro with 4kb of memory

Reply:
No, by my 'logic' (I was asking a goddamn question.) redundant info in docstrings is a waste of space. And I'm not talking about memory >_<.